### DISCUSSION TITLE
Pi-Hole Instructions Clarification

### DISCUSSION THREAD

**Discussion by emorgoch** [blocklistproject/Lists#729] [Q&A]
Under the Pi-Hole instructions, it states "Copy the link to the Pi-hole format for the desired list (from the appropriate table below)". I'm guessing that the "Original" format in the link table is the Pi-Hole format. Maybe that could be made a bit more clear?

**Comment by n0svc** [comment]
Not really meant as an answer, but an add-on. I believe that Pi-hole, starting with FTL v.5.22 on, now supports Adblock Plus (ABP) style lists. 

Tested by adding 2 "AdGuard" version lists. Pi-hole accepted all entries except ones that started with '@@' which I suspect are entries under review because they have "! Issue ###" after the address.

**Reply by iam-py-test** [reply]
> Pi-hole [...] now supports Adblock Plus (ABP) style lists

It does, but it's support is a bit buggy.
Thanks

**Comment by AxiomOperator** [comment]
Thank you for the suggestion. I will look into the recommended list type for Pi-Hole and make it more clear

**Comment by AxiomOperator** [comment]
Done
